swift/ql/src/queries/Security/CWE-079/UnsafeWebViewFetch.qhelp

```diff
@@ -3,6 +3,7 @@
   "qhelp.dtd">
 <qhelp>
 <overview>
+
```

Comment:
On line 12, is the capitalization correct? (web view instead of WebView?) If we are talking about different things, please ignore this comment.

Reply:
Based on https://developer.apple.com/documentation/uikit/uiwebview I think "web view" is correct in prose whereas "UIWebView" and "WKWebView" describe specific classes in code.  So I've standardised on "web view" in the doc.